dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportServiceImpl.java

```diff
@@ -159,6 +172,10 @@
     protected boolean useWorkflow = false;
     protected boolean useWorkflowSendEmail = false;
     protected boolean isQuiet = false;
+    protected boolean processRelationships = false;
```

Comment:
* The rationale for a plain text file was that the items being linked to don't exist yet, since it refers to items in the current import batch. I suppose we could put special values in metadata_relation.xml, like 'folderName:item_001' instead of the linked item UUID, and do an extra pass to update those values to real UUIDs before processing that file. I find a simple manifest file simpler though.
* I completely overlooked that! Will fix.
* Yes, it didn't work for me either. I could fix that, since I can see what the problem is.

Reply:
Fixed test mode and also fixed processing of collection parameter